package.json

```diff
@@ -151,18 +151,18 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "coveralls": "^3.0.1",
-    "eslint": "^6.4.0",
+    "eslint": "^6.8.0",
     "eslint-config-google": "^0.14.0",
-    "eslint-config-prettier": "^6.3.0",
-    "eslint-plugin-jsdoc": "^17.1.0",
-    "eslint-plugin-prettier": "^3.0.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-prettier": "^3.1.0",
     "firebase": "^7.1.0",
     "firebase-admin": "^8.6.1",
     "firebase-functions": "^3.1.0",
     "mocha": "^5.0.5",
     "nock": "^9.3.3",
     "nyc": "^14.0.0",
-    "prettier": "1.14.3",
+    "prettier": "^1.19.0",
```

Comment:
Of note: `prettier` used to not have a `^` there (so it was pinned to a version), but `package-lock.json` should do functionally the same thing, so I don't think it's worth removing.